.github/workflows/ci.yaml

```diff
@@ -0,0 +1,59 @@
+name: CI
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+jobs:
+  cancel-old-workflows:
```

Comment:
Why do we need it?

Reply:
Because the helm chart testing takes few minutes, if you push multiple commits, you don't want to run this test many times, so it just cancels all previous workflows from the same branch when new commits arrive.